addons/mass_mailing/models/mass_mailing.py

```diff
@@ -304,6 +303,18 @@ def _get_mailing_model(self):
     # indirections for inheritance
     _mailing_model = lambda self: self._get_mailing_model()
```

Comment:
By the way, this is old-API style. You can give a *method name* as the selection of a selection field. It works just like the parameter `compute`: it takes into account the overriding of that method.

See https://github.com/odoo/odoo/blob/10.0/odoo/addons/test_new_api/models.py#L208 for instance.